packages/embed/src/components/App.module.css

```diff
@@ -1,39 +1,11 @@
-@font-face {
-  font-family: "Avenir Next LT Pro";
```

Comment:
interesting, we don't need these anymore?

Reply:
just added in other fonts.css file. I still have to sort out some strange font thing here, but I wanted to make sure the layout was structurally the same as packages/web